### PR TITLE
added reconnectNow to allow reconnecting immediately

### PR DESCRIPTION
**Pull request by de-git**
The current reconnect method is just closing the socket and waiting for the reconnect timeout to happen.
I've added a new reconnectNow method that reconnects 'immediately'.
It also can be used to connect if the connection is currently closed.

Both could also happen within the reconnect method instead but I wasn't sure whether the existing behaviour was intentional.

### CODE REVIEW

File: test/unit/angularVertxbusAdapterSpec.js [hunk 1]
```diff
@@ -112,6 +116,61 @@ describe('knalli.angular-vertxbus', function () {
       });
     });
 
+    describe('reconnectNow()', function () {
+      it('should be a function', function () {
+        expect(typeof vertxEventBus.reconnectNow).to.be('function');
+      });
+      it('should call the onopen function if not previously connected', function (done) {
```
Comment (knalli) [inline test/unit/angularVertxbusAdapterSpec.js]: Line is too long is not as expected. It's okay!

**Comment by knalli** [comment]
@de-git  Thank you for the PR.

Please pay attention to the contribution guideline and be branched by and targeted towards `canary´ always. As well as a proper commit message (for the changelog)

**Comment by knalli** [comment]
@de-git  Ehm.. are you still wanting to pull request this here? I'm a little bit confused, i.e.  7e90835 ?!

**Comment by de-git** [comment]
Yes, sorry. Didn't receive a notification email. I will review the contribution guideline and make the necessary changes.

**Comment by de-git** [comment]
This pull request has now been superseded and can be closed.

**Pull request closed without merging**